webpack.config.js

```diff
@@ -3,21 +3,19 @@ const webpack = require('webpack');
 
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
-const PRODUCTION = (process.env.NODE_ENV === 'prod');
-const TEST = (process.env.NODE_ENV === 'test');
+const PRODUCTION = process.env.NODE_ENV === 'prod';
+const TEST = process.env.NODE_ENV === 'test';
 const CG_STYLE_PATH = process.env.CG_STYLE_PATH;
-const CF_SKIN = process.env.CF_SKIN || 'cg';
+
+process.env.SKIN_NAME = process.env.SKIN_NAME || 'cg';
```

Comment:
You can make this as a const variable like the rest of the variables instead of overwriting `process.env.SKIN_NAME`.

In regards to your comment in the PR description, you make it globally available via `webpack.DefinePlugin` down below.

Reply:
@jcscottii by the way, the significance of setting directly into `process.env.SKIN_NAME` is that we want to treat the non-existence of the env var as if it was set to `cg`. This is because we always have to have a skin. I never want to get to a point in the codebase where I have to do `process.env.SKIN_NAME || "cg"`. Thus to avoid this, at the very top we make sure it's always set to something.

With that, I've changed to this:

```js
process.env.SKIN_NAME = process.env.SKIN_NAME || 'cg';
const { SKIN_NAME } = process.env;
```

However, the local var is probably a bit redundant because `process.env.SKIN_NAME` is pretty short as it is. Either way, I've made the change.

Reply:
@jonathaningram i'm just wondering if we can set it once via the `DefinePlugin` that we set below. more docs [here](https://webpack.js.org/plugins/define-plugin/#usage). If it works like i think, once it's there, we don't need to keep redefining the default skin in process.env because it's globally available. BUT i'm not sure if I'm missing something. that's why I asked:

> is there no way for i18next to use the globally exposed variable exposed via the DefinePlugin function below?

the only reason, I can anticipate this not working is if DefinePlugin doesn't take affect by the time `i18next` does it thing.
if i'm correct, you can get rid of those two lines and just have: `const SKIN_NAME = process.env.SKIN_NAME || 'cg';`
if i'm incorrect and i18next can't use the global from DefinePlugin, what you had beforehand was better (now, like you mention, it's redundant).